frontend/src/pages/_app.tsx

```diff
@@ -21,9 +30,30 @@ import 'leaflet/dist/leaflet.css';
 
 // Project styles
 import 'views/styles/index.scss';
-import { getRouterParam } from 'utilities';
 
 const { NEXT_PUBLIC_TRANSIFEX_TOKEN } = process.env;
+const REACT_TOUR_OPTIONS: Omit<MapTourProviderProps, 'children'> = {
+  steps: TOUR_STEPS,
+  padding: 4,
+  showDots: false,
+  className: 'map-tour-popover',
+  maskClassName: 'map-tour-mask',
+  badgeContent: ({ currentStep, totalSteps }) => (
```

Comment:
Same here. Maybe we can put the labels in a different function and load them on the component